troposphere/static/js/components/projects/detail/resources/tableData/instance/Name.react.js

```diff
@@ -15,7 +15,7 @@ export default React.createClass({
       var instance = this.props.instance,
           name = instance.get('name').trim() || "[no instance name]";
 
-      if (!instance.id) {
+      if (!instance && !instance.get('id')) {
```

Comment:
I actually corrected this - but after the merge ... and then I did a horrible thing and _amended_ it to the final commit:

```javascript
       if (instance && !instance.get('id')) {
          return (
            <span>{instance.get('name')}</span>
          );
      }
```

https://github.com/iPlantCollaborativeOpenSource/troposphere/commit/4240c49598d8bf215c61feb67451cd96feb5fe18#diff-40110971ba11f5f70701b6af72b7f17aR18